.github/docker/minimal-nav1-bringup/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG ROS_DISTRO=noetic
+FROM docker.io/ros:$ROS_DISTRO-ros-base
+
+RUN apt update && apt install -y curl ros-$ROS_DISTRO-turtlebot3-navigation ros-$ROS_DISTRO-dwa-local-planner
+
+RUN mkdir -p /tb3 && cd /tb3 \
+  && curl -sL https://github.com/ros-navigation/navigation2/archive/refs/heads/master.tar.gz -o navigation2.tar.gz \
```

Comment:
It looks like this fetches from `master` instead of from [a specific tag](https://github.com/ros-navigation/navigation2/tags). That means the docker file could start failing for users if something is changed upstream. Is that intentional? I can imagine good arguments for this and against this so I don't have a strong opinion on which is better; I just want to make sure the concern was considered.

Reply:
Also if you do want to track the branch instead of a tag you should probably change `master` to `main`. It seems that github is automatically redirecting `master` to refer to `main`, but there's no telling how long they'll keep doing that.

Reply:
Thanks for catching that, that must have been a copy-paste mistake. It definitely makes sense to track a specific tag, since all we need is the map scan. Change to use a tag in https://github.com/open-rmf/free_fleet/pull/145/commits/cccb4033c024b5035a97c7cda146f70970b24baf